src/main/webapp/script.js

```diff
@@ -108,7 +109,9 @@ function createResultCard(name, address, photos, rating, totalCardCount) {
   const websiteButton = document.createElement('button');
   websiteButton.className = 'results-website-button';
   websiteButton.innerText = 'Visit Website';
-  // TODO: Add website link
+  websiteButton.addEventListener('click', function() {
+    window.open(website);
+  });
```

Comment:
nit: I might add a comment here noting that this is equivalent to onclick in html, just to provide a bit more clarity.

Reply:
Done